.travis.yml

```diff
@@ -7,16 +7,24 @@ branches:
     - /^\d\.\d+\.\d+(rc\d+|\.dev\d+)?$/
 matrix:
   include:
+    - env: TOXENV=py 
+      python: 3.9-dev
+      dist: bionic
+    - env: TOXENV=extra-deps
+      python: 3.9-dev
+      dist: bionic
+    - env: TOXENV=asyncio
+      python: 3.9-dev
+      dist: bionic
     - env: TOXENV=security
-      python: 3.8
+      python: 3.9-dev
     - env: TOXENV=flake8
-      python: 3.8
-    - env: TOXENV=pylint
-      python: 3.8
+      python: 3.9-dev
+    - env: TOXENV=typing
+      python: 3.9-dev
```

Comment:
Oops, it looks like you accidentally removed the `pylint` job. Please restore it.